src/Features/CSharp/Portable/UseIsNullCheck/CSharpUseIsNullCheckForReferenceEqualsCodeFixProvider.cs

```diff
@@ -28,17 +28,22 @@ private static SyntaxNode CreateIsNullCheck(SyntaxNode argument)
                 (ExpressionSyntax)argument,
                 SyntaxFactory.ConstantPattern(SyntaxFactory.LiteralExpression(SyntaxKind.NullLiteralExpression))).Parenthesize();
 
-        private static SyntaxNode CreateIsNotNullCheck(SyntaxNode notExpression, SyntaxNode argument)
-            => ((PrefixUnaryExpressionSyntax)notExpression).WithOperand((ExpressionSyntax)CreateIsNullCheck(argument));
+        private static SyntaxNode CreateIsNotNullCheck(SyntaxNode argument)
```

Comment:
Consider renaming method.

Reply:
consider inlining method.

Reply:
> Consider renaming method.

I actually like the current name so decided to leave it.

> consider inlining method.

 This could go either way but decided to leave it since it doesn't seem like a meaningful win to change it.

Reply:
WFM